src/istgt_iscsi_xcopy.h

```diff
@@ -59,12 +62,13 @@ typedef struct istgt_xcopy_tgt_t {
 } ISTGT_XCOPY_TGT;
 
 int istgt_lu_disk_receive_copy_results(CONN_Ptr conn, ISTGT_LU_CMD_Ptr lu_cmd);
-int istgt_lu_disk_xcopy(ISTGT_LU_DISK *spec, CONN_Ptr conn, ISTGT_LU_CMD_Ptr lu_cmd);
-int istgt_lu_disk_process_xcopy(ISTGT_LU_DISK *spec, CONN_Ptr conn, ISTGT_LU_CMD_Ptr lu_cmd, uint8_t *data_buf, int dlen);
-int istgt_lu_disk_lbxcopy(ISTGT_XCOPY_TGT *src_tgt, ISTGT_XCOPY_TGT *dst_tgt, CONN_Ptr conn,
-		ISTGT_LU_CMD_Ptr lu_cmd, int dc, int cat,  uint64_t num_blks_byts, uint8_t sd_opcode);
+int istgt_lu_disk_xcopy(ISTGT_LU_DISK *spec, \
```

Comment:
we don't need 'slashes' in function definitions.. similarly, at other places of function definitions